hosts/hetzner-ephemeral-alpha/default.nix

```diff
@@ -0,0 +1,69 @@
+{ pkgs, config, inputs, lib, ... }:
+{
+  # User options
+  users = {
+    juuso.authorizedKeys = [ "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBNMKgTTpGSvPG4p8pRUWg1kqnP9zPKybTHQ0+Q/noY5+M6uOxkLy7FqUIEFUT9ZS/fflLlC/AlJsFBU212UzobA= ssh@secretive.sandbox.local" ];
+    kari.authorizedKeys = [ "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIKEdpdbTOz0h9tVvkn13k1e8X7MnctH3zHRFmYWTbz9T kari@torque" ];
+  };
+
+  # Users group
+  users.groups.users = { };
+
+  # Localization
+  networking.hostName = "hetzner-ephemeral-alpha";
+  time.timeZone = "Europe/Helsinki";
+
+  # Automatic carbage collection
+  nix.gc = {
+    automatic = true;
+    dates = "weekly";
+    options = "--delete-older-than 30d";
+  };
+
+  # Hercules CI
+  services.hercules-ci-agent = {
+    enable = true;
+    settings.baseDirectory = "/var/mnt/.config/hercules-ci-agent";
+  };
+
+  systemd.mounts = [
+    {
+      enable = true;
+
+      what = "/dev/disk/by-label/nix";
+      where = "/var/mnt/.config";
+      type = "btrfs";
+      options = "subvolid=257";
+
+      before = [ "hercules-ci-agent.service" ];
+      wantedBy = [ "multi-user.target" ];
+    }
+  ];
+
+  systemd.services.linger.enable = lib.mkForce false;
```

Comment:
This line is added because by default linger is enabled for user core globally, even though such user does not exist on this node. I think linger should be moved to be part of the home manager configuration of user core.